hcl2template/types.packer_config.go

```diff
@@ -228,7 +232,8 @@ func (c *PackerConfig) evaluateLocalVariable(local *LocalBlock) hcl.Diagnostics
 		return diags
 	}
 	c.LocalVariables[local.Name] = &Variable{
-		Name: local.Name,
+		Name:      local.Name,
+		Sensitive: local.Sensitive,
```

Comment:
Here `parseLocalVariables` will be called for each file, and it looks like locals is reset/empty for each new time.